src/main/webapp/dev/js/main/instructorFeedbackEdit.es6

```diff
@@ -387,6 +388,78 @@ function backupQuestion(questionNum) {
                                        || $(`#questionTable-${questionNum} > .panel-body`).html();
 }
 
+function isQuestionHavingResponses(questionNum) {
+    const EDIT_STATUS = $(`#form_editquestion-${questionNum}`).attr('editstatus');
+
+    return EDIT_STATUS === 'hasResponses' || EDIT_STATUS === 'mustDeleteResponses';
+}
+
+function getDestructiveFields(questionNum, clonejQueryObject = false) {
+    const $fields = $(`#form_editquestion-${questionNum}`)
+                     .find(':input:enabled')
+                     .not('button')
+                     .not('.nonDestructive')
+                     .not('input[name^="questiondescription"]')
+                     .not('.visibilityCheckbox');
+
+    if (clonejQueryObject) {
+        return $fields.clone();
+    }
+
+    return $fields;
+}
+
+/**
+ * Checks if any changes were made to the destructive fields.
+ * Returns true if changes were made, else false.
+ * @param questionNum
+ */
+function isDestructiveFieldsModifed(questionNum) {
+    const $curr = getDestructiveFields(questionNum, true);
+    const $prev = destructiveFieldsBackup[questionNum];
+
+    if ($curr.length !== $prev.length) {
+        return true;
+    }
+
+    const LENGTH = $prev.length;
+
+    for (let i = 0; i < LENGTH; i += 1) {
+        const prevVal = $($prev.get(i)).val();
+        const currVal = $($curr.get(i)).val();
```

Comment:
I've tried using `.is()`, it doesn't seem to be working. Using `.val()` is giving the expected result.